src/main/java/mServer/crawler/sender/ard/ArdVideoDeserializer.java

```diff
@@ -101,37 +151,11 @@ private UrlWidthHeightDTO deserializeUrlInfos(JsonObject stream) {
         return dto;
     }
     
-    private static String addMissingHttpPrefixIfNecessary(String aUrl) {
-        if(aUrl.startsWith("//")) {
-            aUrl = TEXT_START_HTTP + ":" + aUrl;
-        }
-
-        return aUrl;
-    }
 
-    private static Qualities convertQuality(String quality, UrlWidthHeightDTO urlInfos) {
-        switch(quality) {
-            case "1":
-                return Qualities.SMALL;
-            case "2":
-                return Qualities.NORMAL;
-            case "3":
-                // Beste Qualität, aber nicht immer ist die Auflösung auch HD
-                // Die Prüfung auf den Dateinamen der URL filtert noch zusätzlich
-                // Filme heraus, die eine Auflösung von 960x... haben
-                if ((urlInfos.width == 0 && urlInfos.height == 0 && !urlInfos.url.substring(urlInfos.url.lastIndexOf("/") + 1).startsWith("960"))
-                    || (urlInfos.width >= 1280 && urlInfos.height >= 720)) {
-                    return Qualities.HD;
-                } else {
-                    return Qualities.NORMAL;
-                }
-        }
-        return null;
-    }
-    
     // Hilfsklasse für Zwischenspeichern von URL und zugehörigen Auflösungsinfos
-    private class UrlWidthHeightDTO {
-        public String url = "";
+    private class QualityUrlWidthHeightDTO {
+      public String quality = "";
+        public List<String> urls = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make urls a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)